docs/user-manual/versions.adoc

```diff
@@ -309,7 +324,7 @@ Most existing customisations to the old configuration files and scripts will be
 As such you should compare the old configuration files with the refreshed ones and then port any missing customisations you may have made as necessary.
 The upgrade command itself will copy the older files it changes to an `old-config-bkp.` folder within the instance directory.
 
-Similarly, if you had customised the old `logging.properties` file you may need to prepare analogous changes for the new `log4j2.properties` file.
+Similarly, if you had customised the old `logging.properties` file you may need to prepare analogous changes for the new `log4j2.properties` and `log4j2-utility.properties` files.
```

Comment:
Not sure if this addition makes sense, as this section is about 2.27.0 which wouldnt have that file.

Reply:
I agree with you, I'll revert this change.